Utils/azuretools-core/src/com/microsoft/azuretools/enums/ErrorEnum.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ *
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+ * documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+ * to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all copies or substantial portions of
+ * the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO
+ * THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.microsoft.azuretools.enums;
+
+/**
+ * Enums of backend errors for azure tools.
+ *
+ * @author qianjinshen
+ * @date 2020-09-24 15:05:00
+ */
+public enum ErrorEnum {
+    UNKNOWN_HOST_EXCEPTION(100000, "Encountered an unknown host exception."),
+    SOCKET_TIMEOUT_EXCEPTION(100002, "Encountered a socket timeout exception."),
+    FAILED_TO_GET_ACCESS_TOKEN_BY_CLI(100003, "Failed to get access token by azure cli command."),
```

Comment:
nil, I suppose the message will be shown to users, so `Azure CLI`

Reply:
these messages are only for develop debugging. these messages shown to users lie on ErrorUIMessageEnum.class.

Reply:
anyway, Azure CLI is better to us as well. 👍